samigo/samigo-app/src/webapp/jsf/author/authorSettings.jsp

```diff
@@ -72,6 +72,39 @@
           // adjust the height of the iframe to accomodate the expansion from the accordion
           $("body").height($("body").outerHeight() + 900);
 
+          checkNav = function() {
+              QuesFormatRadios = ["assessmentSettingsAction\\:assessmentFormat\\:0", "assessmentSettingsAction\\:assessmentFormat\\:1", "assessmentSettingsAction\\:assessmentFormat\\:2"];
+
+              if ($("#assessmentSettingsAction\\:itemNavigation\\:0:checked").val() !== undefined) {
+                  enabled = false;
+              } else {
+                  enabled = true;
+              }
+
+              if (enabled) {
+                  QuesFormatRadios.forEach( function(v, i, a) {
+                      $('#assessmentSettingsAction\\:markForReview1').removeAttr("disabled");
+                      $('#assessmentSettingsAction\\:markForReview1').parent().css("color", "#333");
+                      $('label[for="' + v + '"]').css("color", "#333");
+                      $('label[for="assessmentSettingsAction\\:assessmentFormat"]').css("color", "#333");
```

Comment:
is there a class to use instead of color?

Reply:
Not sure, but I can create one if necessary.